ConnectorGrasshopper/ConnectorGrasshopperShared/SchemaBuilder/CreateSchemaObject.cs

```diff
@@ -604,9 +605,17 @@ private object ConvertType(Type type, object value, string name)
       {
         return Convert.ChangeType(value, type);
       }
-      catch (Exception e)
+      catch (InvalidCastException e)
```

Comment:
Would we prefer to use a `when` cause here instead (like elsewhere?)

```cs
catch (Exception ex) when (ex is InvalidCastException or FormatException or OverflowException)
```

Not that this is bad, I'm more curious if there's a reason you've used this pattern here, and the `when` pattern elsewhere

Reply:
Resolved but the comment still stands